admin/class-boldgrid-backup-admin-auto-updates.php

```diff
@@ -133,7 +139,7 @@ public function maybe_update_theme( $stylesheet ) {
 		$theme        = $this->themes->getFromStylesheet( $stylesheet );
 		$theme->setUpdateData();
 		$days_since_release   = $theme->updateData->days; //phpcs:ignore WordPress.NamingConventions.ValidVariableName
-		$theme_update_enabled = isset( $this->settings['themes'][ $stylesheet ] ) ? (bool) $this->settings['themes'][ $stylesheet ] : (bool) $this->settings['plugins']['default'];
+		$theme_update_enabled = isset( $this->settings['themes'][ $stylesheet ] ) ? (bool) $this->settings['themes'][ $stylesheet ] : (bool) $this->settings['themes']['default'];
```

Comment:
Yeah, mixing themes / plugins does seem like it would cause an issue... You haven't been able to replicate, right? IE this PR has no testing instructions?

Reply:
This is correct. There are no real testing instructions for this. I Changed this line because it should have been $this->settings['themes'] all along. I just do not believe we have gotten an error from that particular line yet